### PR TITLE
Add playmoss

### PR DESCRIPTION
**Pull request by 2m**

### CODE REVIEW

**Comment by inverse** [comment]
Thanks for contributing - could you fix the grunt issues then we should be good to go!

**Comment by 2m** [comment]
Fixed the spaces/tabs grunt issue.

**Comment by f1u77y** [comment]
you can just define `Connector.artistTrackSelector` since artist and track are in the same element

**Comment by inverse** [comment]
I'd have to agree with @f1u77y on that one. Would make things simpler

**Comment by 2m** [comment]
I have made the changes requested.

File: connectors/v2/playmoss.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+/* global Connector */
+
+Connector.playerSelector = '#footer-player';
+
+Connector.artistTrackSelector = '#footer-player li.playing a';
+
+Connector.playButtonSelector = 'button.play';
+
+Connector.getArtist = function() {
```
Comment (inverse) [inline connectors/v2/playmoss.js]: You can remove this function

File: connectors/v2/playmoss.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	var text = $(this.artistTrackSelector).text().split(' - ')[0].trim();
+	return text || null;
+};
+
+Connector.getTrack = function() {
```
Comment (inverse) [inline connectors/v2/playmoss.js]: This is no longer required either since the work is done in the BaseConnector via the artistTrackSelector property

**Comment by 2m** [comment]
Ahh thanks. Did not know that there is split by hyphen functionality already build in.

**Comment by inverse** [comment]
Super :+1: Thanks for the commit :)

**Pull request merged**